firestore-send-email/extension.yaml

```diff
@@ -181,6 +181,28 @@ params:
       if the template is specified in the added email document.
     type: string
 
+  - param: FIRESTORE_EXPIRE_AT
+    label: Firestore TTL support option
+    description: >-
+      Do you want the firestore records to be marked with an expireAt field for a TTL policy? If so, how long do you want the records to stay ineligible for deletion by
+      your policy? Note you will have to set up the [TTL policy](https://firebase.google.com/docs/firestore/ttl) for your project yourself. If "Never" is selected then no
+      expireAt field will be added.
+    type: select
```

Comment:
is this the best DX? maybe integer number of hours is better for the developer, as it's more flexible